docs/source/widgets.rst

```diff
@@ -851,6 +853,163 @@ More customization options on ``Batch`` class:
   rendering next page link.
 
 
+Batched Items
+-------------
+
+A tile for creating batched, searchable listings is contained at
+``cone.app.browser.batch``.
+
+It consists of a listing header which displays a search field and a slice size
```

Comment:
The plain pagination batch is implemented at ``cone.app.browser.batch.Batch`` (the name Batch might change to ``Pagination`` in future since it seems it's a browser used terminus for what it actually does) while ``BatchedItems`` represents a full blown view for displaying a slice of items related to a pagination page. The contained search field is there for pragmatic reasons. Most of the time a user wants to be able restrict the available overall items anyways.